### PR TITLE
Support explicit operation for when to load full resource table

### PR DESCRIPTION
**Pull request by sv99**
Explicit loading restable for support skip resource decoding

### CODE REVIEW

File: brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
 import brut.directory.ExtFile;
 import brut.directory.FileDirectory;
 import brut.util.Duo;
+import com.sun.org.apache.xpath.internal.operations.And;
```
Comment (iBotPeaches) [inline brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java:33]: Looks like a bad import

File: brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java [hunk 2]
```diff
@@ -85,9 +82,9 @@ public ResTable getResTable() throws AndrolibException {
             throw new AndrolibException(
                 "Apk doesn't contain either AndroidManifest.xml file or resources.arsc file");
         }
-        if (hasResources() && !mResTable.isMainPkgLoaded()) {
-            mResTable.loadMainPkg(mApkFile);
-        }
+//        if (hasResources() && !mResTable.isMainPkgLoaded()) {
+//            mResTable.loadMainPkg(mApkFile);
+//        }
```
Comment (iBotPeaches) [inline brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java:87]: commented

File: brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java [hunk 3]
```diff
@@ -191,22 +181,14 @@ private ExtMXSerializer getResXmlSerializer() {
...
-                    } catch (DirectoryException ex) {
-                        throw new AndrolibException(ex);
-                    }
-                    break;
-                case Config.DECODE_RESOURCES_FULL:
-                    decodeResources(getResTable(), mApkFile, outDir);
-                    break;
-            }
-            mResTable.initApkInfo(mApkInfo, outDir);
+            load();
```
Comment (iBotPeaches) [inline brut.apktool/apktool-lib/src/main/java/brut/androlib/res/ResourcesDecoder.java:190]: We might want a more descriptive function name. If it just wraps `loadMainPkg`

**Review by iBotPeaches — CHANGES_REQUESTED** [review]

**Comment by sv99** [comment]
Corrected

**Review by iBotPeaches — APPROVED** [review]

**Pull request merged**